programs/pawn-shop/src/lib.rs

```diff
@@ -329,18 +289,20 @@ pub mod pawn_shop {
             pawn_loan.status = LoanStatus::Defaulted;
             pawn_loan.end_time = unix_timestamp;
 
+            // Thaw token account then transfer to lender
+            thaw_pawn_token_account!(ctx);
```

Comment:
After thaw, should we revoke the token account?

Reply:
In the seize case we don't want to revoke because we need to transfer it to the lender.
Not sure it is worth it to make a helper for 2/3 cases.